cmd/marker/main.go

```diff
@@ -74,11 +84,27 @@ func main() {
 		glog.Fatalf("Failed to create a new marker object: %v", err)
 	}
 
-	for {
-		err := markerApp.Update()
+	markerCache := cache.Cache{}
+	wait.JitterUntil(func() {
+		jitteredReconcileInterval := wait.Jitter(time.Duration(*reconcileInterval)*time.Minute, 1.2)
+		shouldReconcileNode := time.Now().Sub(markerCache.LastRefreshTime()) >= jitteredReconcileInterval
+		if shouldReconcileNode {
+			reportedBridges, err := markerApp.GetReportedResources()
+			if err != nil {
+				glog.Errorf("GetReportedResources failed: %v", err)
+			}
+
+			if !reflect.DeepEqual(markerCache.Bridges(), reportedBridges) {
+				glog.Warningf("cached bridges are different than the reported bridges on node %s", *nodeName)
+			}
+
+			markerCache.Refresh(reportedBridges)
+		}
+
+		err := markerApp.Update(&markerCache)
 		if err != nil {
 			glog.Fatalf("Update failed: %v", err)
 		}
-		time.Sleep(time.Duration(*pollInterval) * time.Second)
-	}
+
+	}, time.Duration(*updateInterval)*time.Second, 1.2, true, wait.NeverStop)
```

Comment:
Any chance you did not run `go fmt` on this (surprised about different `*` formatting)?

Reply:
Actually I did. I also re-checked and it stays in tact. Would you like me to change that? (this kind of formatting might also be needed in [marker_test](https://github.com/k8snetworkplumbingwg/ovs-cni/blob/main/tests/marker_test.go#L64) )

Reply:
If both are compatible, it's all good. Thanks for checking